src/arrays/delegate-tasks.js

```diff
@@ -1,177 +1,615 @@
 import Ant from '../images/ant.png';
 import PrimU from '../images/prim-u.png';
 import avatar from '../images/review-avatar.png';
+import graph from '../images/data-layers.png';
+import wordCloud from '../images/word-cloud.png';
+import flumeScience from '../images/flume-science.png';
+import connectedWorld from '../images/connected-world.png';
 
 export const webDevProjects = [
   {
     id: 1,
-    'top-right': {
+    image: {
       image: Ant,
     },
-    'top-left': {
+    info: {
       title: 'Ant Robotics',
       subtitle:
         'Landing page website for the company that cakes automated logistics robots from Oakland, CA',
       date: 'May 2021',
-      link: 'Visit wbesite →',
-    },
-    'bottom-right': {
-      avatar,
-      title: 'Maxim Antonenko',
-      subtitle: 'Ant Robotics CEO',
-      description:
-        '“The Data Analytics students have highlighted a few languages which show a growing demand for translation and expanded our list of keywords to detect more translation-related user reviews,” Demid said. “The Data Science students tested a wide range of natural language processing algorithms, including deep machine learning..."',
-      link: 'Read full review',
-    },
-    'bottom-left': {
-      avatars: [avatar, avatar, avatar],
-      title: 'Students team',
-      subtitle: 'Maria Wright, Maria Wright, Enyel Sequeira',
-      description:
-        '“I learned so much from it,” Xia said. “Not just about new packages and methods of analysis and reporting, but also workflow management, how to clarify task requirements, collaborate with others, and more. Those are the kind of experiences I can put on my resume, and that’s awesome too.”',
+      link: 'https://www.example.com/',
```

Comment:
Totally forgot that links also need a link. Thank you for adding this!

Reply:
Of course! I omitted linkText for the longest time until I realized that the the link-text depends on the curriculum, so thank _you_ for adding that portion. Seriously, I cannot overstate how nice it was to have this data all set and ready to go in such a beautiful manner, you rock.